lib/modules/datasource/docker/index.spec.ts

```diff
@@ -35,6 +39,9 @@ function mockEcrAuthReject(msg: string) {
 
 describe('modules/datasource/docker/index', () => {
   beforeEach(() => {
+    if (GoogleAuth) {
```

Comment:
why the if?

Reply:
without the if I got errors saying it was undefined (before any of the tests using it ran)

Reply:
With the updated tests I no longer need this if statement